db/migrate/20200901085205_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :comments do |t|
+      t.string :title
+      t.text :body
+      t.integer :author_id
```

Comment:
- [ ] Also this too make a migration to change it to `user_id`.

Reply:
It's already user_id. I believe you are looking at an old commit.